lib/models.js

```diff
@@ -802,6 +802,10 @@ class Constraint {
       "lastModifiedBy": (this.lastModifiedBy) ? this.lastModifiedBy.fqn : undefined,
     }
   }
+
+  toString() {
+    return this.constructor.name;
```

Comment:
I realize this is just a fallback in case the constraint subclass doesn't define `toString`, but if there is a `path`, please represent it in `toString`.